src/components/graph-component/graph.component.js

```diff
@@ -0,0 +1,186 @@
+import { graphWrapperCanvas, canvasBoard, toggleForGraph, selectOptions } from "./graph.template";
+import { CovidDashboardService } from "../../core/services/index";
```

Comment:
It should be import `{ CovidDashboardService, COLOR_PALETTE } from "../../core/index";`